modules/460-log-shipper/hooks/internal/vector/transform/testdata/extra-labels.json

```diff
@@ -1,9 +1,7 @@
 [
 	{
 		"drop_on_abort": false,
-		"inputs": [
-			"testit"
-		],
+		"inputs": [],
```

Comment:
? How we can have a transform without an input? Something have changed in the vector?

Reply:
I got that you don't want to have an transform if no rules set. So in that case we have a global transform without inputs and it's just not working, right?

Reply:
Eww, this testdata shows only the part of the config. 

The actual change that caused the inputs array to be empty here is this piece of the code
https://github.com/deckhouse/deckhouse/blob/1d85d1f9e117d9a444265008f9221c71b379bdae/modules/460-log-shipper/hooks/internal/vector/transform/transforms.go#L28-L44

The `prevInput` variable was equal to the `inputName` and always added as the input for all transform maps. This is an error because `inputName` is never a valid input name. Inputs for transforms were overridden further in the composer's code.